dpnp/tests/test_fft.py

```diff
@@ -926,6 +927,16 @@ def test_rfft_1D_on_2D_array_out(self, dtype, n, axis, norm, order):
         expected = numpy.fft.rfft(a_np, n=n, axis=axis, norm=norm)
         assert_dtype_allclose(result, expected, check_only_type_kind=True)
 
+    @pytest.mark.skipif(not has_support_aspect16(), reason="no fp16 support")
+    def test_float16(self):
+        a = numpy.ones(10, dtype=numpy.float16)
```

Comment:
It seems it would be better to use arange here, otherwise for input array of ones we will get the result with only the first non-zero element.
```suggestion
        a = numpy.arange(10, dtype=numpy.float16)
```